src/dev/flang/fe/LibraryFeature.java

```diff
@@ -868,7 +875,57 @@ else if (other instanceof LibraryFeature lf)
     return result;
   }
 
+  /**
+   * Union of the library features of all inner features. Checks inner features recursively.
```

Comment:
I think in this comment library feature and library module is mixed up?

Reply:
yes